lib/http.js

```diff
@@ -6,13 +6,17 @@
 'use strict';
 
 const async = require('async');
+const {asyncHandler} = require('bedrock-express');
 const bedrock = require('bedrock');
 const brLedgerAgent = require('./api');
 const brLedgerNode = require('bedrock-ledger-node');
 const brPassport = require('bedrock-passport');
 const brRest = require('bedrock-rest');
 const {config, util: {callbackify, BedrockError}} = bedrock;
 const cors = require('cors');
+const {promisify} = require('util');
+
+const brLedgerAgentGet = promisify(brLedgerAgent.get);
```

Comment:
Ideally this wouldn't be necessary anymore with the new libs.

Reply:
Per discussion, this will be addressed in a follow-on PR.